### ISSUE TITLE
npm ERR! cb() never called!

### ISSUE THREAD

**Issue by asonofchina** [npm/npm#20862]
>npm install
npm WARN tar zlib error: unexpected end of file
npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! A complete log of this run can be found in:
npm ERR!     H:\Software\nodejs\node_cache\_logs\2018-06-01T02_30_48_542Z-debug.log

**Comment by legodude17** [comment]
Does this occur in the latest version of npm?

**Comment by TejuNaveen** [comment]
Yes, it is occurring in the latest, I have the npm version 6.1.0, node version 10.3.0 & i'm facing the same issue.
It's occurring for any package which I'm trying to install, be it body-parser or request packages.

**Comment by leochiong** [comment]
How to solve this problem?
